packages/react/src/prefabs/PreJoin.tsx

```diff
@@ -231,17 +230,6 @@ export function PreJoin({
   videoProcessor,
   ...htmlProps
 }: PreJoinProps) {
-  const [userChoices, setUserChoices] = React.useState(defaultUserChoices);
-
-  // TODO: Remove and pipe `defaults` object directly into `usePersistentUserChoices` once we fully switch from type `LocalUserChoices` to `UserChoices`.
```

Comment:
@Ocupe do you remember why this was handled in this particular way? 
Type wise nothing is complaining when simply piping the defaults directly into `usePersistentUserChoices`

Reply:
I think the TODO note says why, right? We had this transition period with different/incompatible types.

Reply:
I think because of the default/fallback parameter for `defaults` (line 220) it will trigger a constant rerender. Maybe it was a way to stop that from happening.

Reply:
but both places are using the same type `LocalUserChoices` 🤔 
not sure where `UserChoices` is defined at all. Maybe this was an oversight when `UserChoices` got removed?